### ISSUE TITLE
Team update - May 04, 2022

### ISSUE THREAD

**Issue by trallard** [Quansight-Labs/jupyter-a11y-mgmt#101]
This is a @Quansight-Labs/czi-a11y-grant team sync 🎉🎉🎉! This is a way for the Team Members to provide status reports on what they've been up to this week and request help and attention for things they are working on. This issue will be closed at the end of the day.

_Copy and paste the template below, and answer questions as you wish!_

<details>
<summary>Response Template</summary>

```md
**Thanks I'd like to give 🙌**
- So-and-so helped me out a lot with foo...

**Updates from last week :heavy_check_mark:**
- I worked towards deliverable: <link-to-deliverable>

**Challenges I faced and things I'd like assistance with 🙏**
- I had a hard time figuring out ...

**My availability for next week**
- I'll be off on foo day...

**Important items for discussion 💬**
- I have a question about goal <link-to-goal>
- Can @foo give a comment on issue #NN ?
- I opened #NN for discussion, please chime in
- I need to discuss foo
```

</details>

#### :mag: Needs Triage

_The [Needs Triage issues](https://github.com/Quansight-Labs/jupyter-a11y-mgmt/labels/status%3A%20needs%20triage) require an initial assessment and labeling._

No issues need triage! 🎉

### :dart: Project boards

Please make sure the boards reflect the current status of the project.

[Jupyter a11y grant project](https://github.com/orgs/Quansight-Labs/projects/5/views/5)

**Comment by gabalafou** [comment]
**Thanks I'd like to give 🙌**
- Thanks to @isabela-pf for the really great work with mapping tests to WCAG guidelines and coming up with test scripts
- Thanks to @tonyfast for setting up Doit and handing off the code to me so I could learn from it
- Thanks to @trallard for helping to reduce complexity with Gitpod

**Updates from last week :heavy_check_mark:**
- I have been diving into the guts of Playwright and Galata to clear away issues we will have when installing them into a dedicated environment (Gitpod)

**Challenges I faced and things I'd like assistance with 🙏**
- I REALLY want to get up a PR agains jupyter/accessibility repo that allows the axe-core tests to run in a Gitpod environment. I'm worried about being blocked on this. Should I work in parallel on my own Gitpod account?

**My availability for next week**
- Available all next week

**Important items for discussion 💬**
- Maybe some shift left discussion about what we're building right now and how we imagine people interacting with it. Some things that came out of conversation with Martha in yesterday's accessibility meeting: 
  - Document, document, document! 
  - How would a developer add an a11y test and then run it? How do we want that to look? What makes sense?

**Comment by isabela-pf** [comment]
**Thanks I'd like to give 🙌**
- Thank you to you all for good feedback on #95!

**Updates from last week :heavy_check_mark:**
- #95. I'm working on an updated version of the testing scripts based on y'all's feedback. It will be posted in that issue, but it's not totally polished as of writing this.
- #3. I also started turning some of the [brainstorm event ideas](https://github.com/isabela-pf/a11y-events/issues/4) into agendas. Nothing to show yet, but that's a next step I'm taking because I think it will be easy to critique agendas.

**Challenges I faced and things I'd like assistance with 🙏**
- Nothing unusual, thanks for asking.

**My availability for next week**
- I'll be here all week.

**Important items for discussion 💬**
- I briefly talked with @gabalafou about #3 and he mentioned he wanted to bring the discussion to you all. The main thoughts were
     - Are events really outreach to the communities we currently fail to engage (they aren't as they are now), as mentioned in the issue?
     - What are ways to meet disability and/or accessibility communities where they are at (with the resources we have)?
